website/src/content/docs/index.mdx

```diff
@@ -20,40 +18,12 @@ hero:
       link: https://github.com/biomejs/biome
       icon: external
       variant: secondary
-banner:
-  content: |
-    Biome claims the bounty of the <a href="/blog/biome-wins-prettier-challenge">Prettier challenge</a>!
 ---
 
+import Support from '../../components/Support.astro';
+import Performance from '../../components/Performance.astro';
+import Cardsinfo from '../../components/Cardsinfo.astro';
 
-import { Card, CardGrid } from '@astrojs/starlight/components';
```

Comment:
Separate code in a component file makes it difficult to support translation, you may need to expose the texts using props or keep the code as it was before the change